### PR TITLE
Sharing

### PR DESCRIPTION
**Pull request by bbergstrom**
Adding support for specifying account ids to share the snapshots with.  This is useful for sharing production snapshots with a pre-production account/environment.

### CODE REVIEW

**Comment by anfleene** [comment]
:+1:

File: files/default/automated-backup.sh [hunk 1]
```diff
@@ -77,6 +77,23 @@ create_EBS_Snapshot_Tags() {
   fi
 }
 
+add_EBS_permissions() {
```
Comment (NickLaMuro) [inline files/default/automated-backup.sh]: So nothing is calling this function.  I would assume this is something that should be called in the main function and this function can determine if it needs to make the aws cli call or not.

Comment (NickLaMuro) [inline files/default/automated-backup.sh]: This also seems like it is supposed to be called after the snapshot is created (though, I assume you knew that).

Comment (bbergstrom) [inline files/default/automated-backup.sh]: YESSSSSS

**Comment by anfleene** [comment]
:+1:

**Pull request merged**